controller.py

```diff
@@ -485,6 +496,12 @@ def delete_embedding(project_id: str, embedding_id: str) -> int:
     org_id = organization.get_id_by_project_id(project_id)
     s3.delete_object(org_id, project_id + "/" + object_name)
     request_util.delete_embedding_from_neural_search(embedding_id)
+    if get_config_value("is_managed"):
```

Comment:
absolute edge case for dev but i don't think we need the manged check in the delete methods since if the path exists it should be deleted anyway. (For the dev who switches between managed and not :) )

Reply:
removed the if-clause